app/Http/Controllers/AssetsController.php

```diff
@@ -137,6 +137,7 @@ public function store(AssetRequest $request)
         $asset->supplier_id             = request('supplier_id', 0);
         $asset->requestable             = request('requestable', 0);
         $asset->rtd_location_id         = request('rtd_location_id', null);
+        $asset->location_id             = request('rtd_location_id', null);
```

Comment:
Weird; this change doesn't seem necessary - below, you have:

```php
if ($asset->assigned_to=='') {
   //etc
```

Reply:
Hm. You're right...

Reply:
Maybe this is only necessary in the API controller.